frontend/packages/knative-plugin/src/components/overview/__tests__/overview-knative-test-data.ts

```diff
@@ -0,0 +1,45 @@
+import { K8sResourceKind } from '@console/internal/module/k8s';
```

Comment:
Sorry I didn't realize we can update  https://github.com/openshift/console/blob/master/frontend/packages/dev-console/src/components/topology/__tests__/topology-knative-test-data.ts instead of creating own data